### PR TITLE
Add NextAuth & fix some fluency errors & more!

### PR DESCRIPTION
**Pull request by leowilkin**
Changes made:
1. Added information about NextAuth .env variables
2. Added callout for Windows testers
3. Fixed some fluency errors

### CODE REVIEW

**Comment by swve** [comment]
I think we can add a step just after the 3rd in 'self-hosting/hosting-guide', so that the user immediately configure nextauth, otherwise he would not be able to sign in

**Review by swve — CHANGES_REQUESTED** [review]
I think we can add a step just after the 3rd in 'self-hosting/hosting-guide', so that the user immediately configure nextauth, otherwise he would not be able to sign in

**Comment by leowilkin** [comment]
Okay - working on that

File: pages/self-hosting/configuration.mdx [hunk 1]
```diff
@@ -142,13 +142,29 @@ Your System Email Address
 
 You can also configure your frontend by creating a `.env` file in the `apps/web` folder.
```
Comment (swve) [inline pages/self-hosting/configuration.mdx:144]:
```suggestion
### Authentication (Oauth) configuration 
```

Comment (swve) [inline pages/self-hosting/configuration.mdx:144]: ### Authentication (Oauth) configuration

Comment (swve) [inline pages/self-hosting/configuration.mdx:144]:
```suggestion
### Authentication (Oauth) configuration
```

File: pages/self-hosting/hosting-guide.mdx [hunk 2]
```diff
@@ -34,6 +38,24 @@ git clone https://github.com/learnhouse/learnhouse
...
+### Configure NextAuth
+
+Add these lines to your `.env` file. A template is available under `/extra/example-learnhouse-conf.env`.
+
+<Callout type="info">
+To safely encrypt tokens and email hashes, you need a NextAuth secret.
+
+To create one, either run `npx auth secret`, or on UNIX based systems (MacOS/Linux) you can use `openssl rand -base64 33`.
+
+Copy this string into your .env file as NEXTAUTH_SECRET={your secret}
```
Comment (swve) [inline pages/self-hosting/hosting-guide.mdx:50]:
```suggestion
Copy this string into your .env file as NEXTAUTH_SECRET=
```

**Review by swve — APPROVED** [review]

File: pages/self-hosting/hosting-guide.mdx (hunk 2 above)
Comment (leowilkin) [reply pages/self-hosting/hosting-guide.mdx]: Won't that error it?

**Comment by leowilkin** [comment]
I'll add the Linode line, will have to find it in the history.

**Comment by leowilkin** [comment]
Closing without Linode

**Pull request merged**